test/test_evaluation/test_test_evaluator.py

```diff
@@ -128,6 +128,34 @@ def test_eval_test(self):
         self.assertEqual(rval[0]["status"], StatusType.SUCCESS)
         self.assertNotIn("bac_metric", rval[0]["additional_run_info"])
 
+    def test_eval_test_multi_objective(self):
+        metrics = {
+            accuracy: 0.040000000000000036,
+            balanced_accuracy: 0.02777777777777779,
+        }
+        eval_t(
+            queue=self.queue,
+            backend=self.backend,
+            config=self.configuration,
+            metrics=list(metrics.keys()),
+            seed=1,
+            num_run=1,
+            scoring_functions=None,
+            output_y_hat_optimization=False,
+            include=None,
+            exclude=None,
+            disable_file_output=False,
+            instance=self.dataset_name,
+            port=self.port,
+            additional_components=dict(),
+        )
+        rval = read_queue(self.queue)
+        self.assertEqual(len(rval), 1)
+        for metric, loss in metrics.items():
+            self.assertAlmostEqual(rval[0]["loss"][metric.name], loss)
+        self.assertEqual(rval[0]["status"], StatusType.SUCCESS)
+        self.assertNotIn("bac_metric", rval[0]["additional_run_info"])
```

Comment:
[Q]
What's this `"bac_metric"` check?

Reply:
TBH I don't recall, I copied most of the test from the single objective test.